lib/models/category.ts

```diff
@@ -28,19 +29,19 @@ export = (ctx: Hexo) => {
     return str;
   });
 
-  Category.virtual('path').get(function() {
+  Category.virtual('path').get(function(this: CategorySchema) {
```

Comment:
Is it possible to change the `warehouse.Schema` type so that the `get` callback always infer `this` as its Schema?

Reply:
I forgot to change this